.vib/grafana/cypress/cypress/integration/grafana_test_suite.js

```diff
@@ -72,84 +71,83 @@ it('checks if smtp is configured', () => {
 
   cy.login();
   cy.visit('admin/settings');
-  cy.contains('td', SMTP_ADDRESS)
-    .should('exist');
-  cy.contains('td', SMTP_NAME)
-    .should('exist');
-  cy.contains('td', SMTP_USER)
-    .should('exist');
-})
+  cy.contains('td', SMTP_ADDRESS);
+  cy.contains('td', SMTP_NAME);
+  cy.contains('td', SMTP_USER);
+});
 
 it('checks if plugin page is showing plugins', () => {
   cy.login();
   cy.visit('/plugins?filterByType=datasource');
-  cy.get('[data-testid*="plugin-list"]')
-    .should('exist');
-})
+  cy.get('[data-testid*="plugin-list"]').should('exist');
```

Comment:
```suggestion
  cy.get('[data-testid*="plugin-list"]');
```

This should be unnecessary, right?

Reply:
I want to check if the plugin list exists. I am not using `cy.contains()`, so I'd prefer to add an assertion so that the test fails with some additional info.

Reply:
Using Cypress 9.5.3, if the element does not exist in the DOM it does not matter whether the assertion is specified or not (the message is the same):

<img width="300" alt="Screenshot 2022-04-11 at 16 28 58" src="https://user-images.githubusercontent.com/24253844/162762393-8e6ebace-8d23-4d5b-93c8-ac91983d153e.png">

<img width="300" alt="Screenshot 2022-04-11 at 16 30 01" src="https://user-images.githubusercontent.com/24253844/162762304-d7053c17-edb0-40e3-a512-8ecc5ddc04aa.png">

Having said that, even if the message is different, IMHO we should add a comment to every `be.visible`/`exists` assertion that appears in the tests to indicate they are there on purpose.

Reply:
OK, removed the assertion and left only `cy.get()`.